x-pack/test/accessibility/config.ts

```diff
@@ -44,7 +45,6 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       require.resolve('./apps/license_management'),
       require.resolve('./apps/tags'),
       require.resolve('./apps/search_sessions'),
-      require.resolve('./apps/stack_monitoring'),
```

Comment:
Do we want to remove it?

Reply:
@jughosta sorry didn't understand. What would you like me to remove? Thanks!

Reply:
This PR removes the unrelated `require.resolve('./apps/stack_monitoring'),`. Checking with you if it was an intended change or not.

Reply:
@jughosta ah. That's merge problems. Thanks @jughosta . Will resolve it.